source/views/dictionary/list.js

```diff
@@ -130,6 +130,7 @@ export class DictionaryView extends React.Component {
         <SearchBar
           getRef={ref => (this.searchBar = ref)}
           onChangeText={this.performSearch}
+          onSearchButtonPress={() => this.searchBar.unFocus()}
```

Comment:
Passing in the function reference instead of creating an anonymous function resulted in errors. In short, the `searchBar` ref was null.